src/zones/cc-zone-input.js

```diff
@@ -0,0 +1,370 @@
+import './cc-zone.js';
+import '../maps/cc-map.js';
+import '../maps/cc-map-marker-server.js';
+import { css, html, LitElement } from 'lit-element';
+import { classMap } from 'lit-html/directives/class-map.js';
+import { repeat } from 'lit-html/directives/repeat.js';
+import { dispatchCustomEvent } from '../lib/events.js';
+import { i18n } from '../lib/i18n.js';
+import { withResizeObserver } from '../mixins/with-resize-observer.js';
+
+const SKELETON_ZONES = new Array(6).fill(null);
+const CLEVER_CLOUD_ZONE = 'infra:clever-cloud';
+const PRIVATE_ZONE = 'scope:private';
+
+/**
+ * A input component to select a zone with a map and a list.
+ *
+ * * 🎨 default CSS display: `grid`
+ * <br>
+ * 🧐 [component's source code on GitHub](https://github.com/CleverCloud/clever-components/blob/master/src/zones/cc-zone-input.js)
+ *
+ * ## Details
+ *
+ * * When `zones` is nullish, a skeleton screen UI pattern is displayed (loading hint).
+ * * Zones are sorted in the list using `tags`. Clever Cloud, then private, the alphanum sort on the city name.
```

Comment:
did you mean `then alphanum`?

Reply:
I meant "classic alphanumeric sort". I fixed that and also replaced the "the" with "then".